lib/data_update_scripts/20210630041322_migrate_data_to_work_field.rb

```diff
@@ -0,0 +1,11 @@
+module DataUpdateScripts
+  class MigrateDataToWorkField
+    QUERY = "data->>'employment_title' <> '' AND data->>'work' IS NULL".freeze
```

Comment:
The `data->>'employment_title' <> ''` part takes care of both `null` and empty strings.

Reply:
Can you clarify this a bit, maybe in a comment in the code? It took me several minutes to understand how this would be true because `NULL` equality rules aren't very intuitive for people like me who don't have math degrees. 🙂

Reply:
One part of it is the JSON operator used (`->>` instead of `->`):

![image](https://user-images.githubusercontent.com/47985/124144714-68acf300-dab6-11eb-846e-5d28cb2ddbca.png)

(`(null)` is what I set in my `~/.psqlrc` as `NULL` representation).

So `->` returns a JSONB `NULL` whereas `->>`  returns a "Postgres `NULL`" 

The other part is that `NULL` compared to a string results in a `NULL`, it's neither empty nor not empty:

![image](https://user-images.githubusercontent.com/47985/124147836-423c8700-dab9-11eb-8f42-540e63ae5bd4.png)

But I have to admit, I always need to remind myself of the `stringexpression <> ''` trick to check for "not null and not empty" and always end up back at https://stackoverflow.com/questions/23766084/best-way-to-check-for-empty-or-null-value.

Reply:
Added a comment to the code.